contracts/minter/src/contract.rs

```diff
@@ -135,6 +135,7 @@ pub fn instantiate(
                 symbol: msg.sg721_instantiate_msg.symbol,
                 minter: env.contract.address.to_string(),
                 collection_info: msg.sg721_instantiate_msg.collection_info,
+                num_tokens: None,
```

Comment:
i don't understand why you're adding this in here. is it to make dynamic collection fees optional?